packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -34,11 +39,33 @@ export function generate(bundle: unknown, source: JSONSchema): Either<Error, unk
     E.chain(updatedSource =>
       // necessary as workaround broken types in json-schema-faker
       // @ts-ignore
-      tryCatch(() => jsf.generate({ ...cloneDeep(updatedSource), __bundled__: bundle }), toError)
+      tryCatch(() => sortSchemaAlphabetically(jsf.generate({ ...cloneDeep(updatedSource), __bundled__: bundle })), toError)
     )
   );
 }
 
+//sort alphabetically by keys
+export function sortSchemaAlphabetically(source: any): any {
```

Comment:
Can you write some unit tests for this function?

Probably with a test case for each of the following:
1. not an object or array --> leaves source untouched
2. array of non-objects (i.e. strings) --> leaves the array untouched
3. array of objects --> sorts each object
4. simple object (i.e. each property has just a string value) --> properties are sorted and values stay with the keys
5. simple complex object (i.e. some properties are just strings, others are objects) --> top level object has properties sorted as well as sub object(s)
6. Maybe a performance-ish test of a bunch of nested objects and arrays and maybe the object has a shit ton of properties to sort?
7. other cases I haven't thought of...?

I think it would also be good to write a [test harness test](https://github.com/stoplightio/prism/tree/master/test-harness#adding-a-new-test) for this so we have an E2E test of this working.